libs/lib_parsing/Tok.ml

```diff
@@ -242,15 +242,22 @@ let content_of_tok_opt ii =
    inspecting the contents of the token, plus the start information.
 *)
```

Comment:
Could have been good to add a comment like (* See Unit_tok.test_end_pos_of_loc to better understand this function by seeing concrete examples *)